deps/rabbitmq_aws/src/rabbitmq_aws.erl

```diff
@@ -497,15 +497,15 @@ sign_headers(#state{access_key = AccessKey,
 %% @doc Determine whether or not an Imdsv2Token has expired.
 %% @end
 expired_imdsv2_token(undefined) ->
-  _ = rabbit_log:debug("EC2 IMDSv2 token has not yet been obtained."),
+  rabbit_log:debug("EC2 IMDSv2 token has not yet been obtained"),
```

Comment:
Please retain the `_ = ` part. It is needed on `v3.8.x` because of Lager parse transforms, they produce a warning if the return value is not explicitly ignored, which breaks compilation on Erlang 24.

Reply:
Done!